Chefs/Views/MainPage.xaml

```diff
@@ -47,7 +47,7 @@
 				</utu:TabBarItem.Icon>
 			</utu:TabBarItem>
 
-			<utu:TabBarItem uen:Region.Name="-/Search"
+			<utu:TabBarItem uen:Region.Name="Search"
```

Comment:
```suggestion
			<utu:TabBarItem uen:Region.Name="-/Search"
```

We still need this otherwise when navigating to Search via TabBar filters aren't cleared